dice.cpp

```diff
@@ -0,0 +1,33 @@
+#include "dice.h"
+
+Dice::Dice(){
+    sum =0;
+}
+
+Dice::~Dice(){}
+
+void Dice::displayResults(){
+    std::cout << "Results of "<< results.size() << "d" << diceSides << " rolls:" << std::endl;
+    for(auto i : results){
```

Comment:
Doing "auto i" will copy each result. Not a problem here, since this is a primitive data type. But FYI, you can do "auto& i" to avoid big copies.